R/interpret.R

```diff
@@ -143,7 +173,14 @@ plot.interpret <- function(x, include_intercept = FALSE, max_char = 40,
   limits <- max(abs(x$coefficient)) * c(-1.05, 1.05)
   the_plot <-
     x %>%
-    ggplot(aes(x = reorder(variable, coefficient), y = coefficient)) +
+    ggplot(aes(x = reorder(
+      ifelse( # Add the reference level if it exists
```

Comment:
To make your code easier to reason about, don't make any but the simplest modifications inside a ggplot call. Make the changes first, then pass the object to ggplot.

Reply:
Good call. Originally I didn't know if you were okay with me manipulating some of the data. I understand the principle now. Thank you.